ZenPacks/zenoss/LinuxMonitor/HardDisk.py

```diff
@@ -80,15 +85,40 @@ def impacted_object(self):
             return lv
 
     def storage_disk_lun(self):
-        # return the UCS storage disk/virtual drive
-        # if disk_ids contains the IDs of disk/virtual drive
+        """Return the storage disk/virtual drive based on disk_ids."""
```

Comment:
The name of this method (singular) is a bit confusing since it's a generator. It's probably not a good idea to rename the method, but the docstring can at least be made to indicate that this is a generator, and isn't going to return a single object.